cfn-resources/test/e2e/utility/aws_helper.go

```diff
@@ -179,3 +197,12 @@ func TestIsTemplateValid(t *testing.T, svc *cfn.Client, template string) {
 	_, err := svc.ValidateTemplate(context.Background(), input)
 	FailNowIfError(t, "invalid cloudformation stack: %v", err)
 }
+
+func DeleteStackIfExists(t *testing.T, c *cfn.Client, stackName string) error {
+	_, err := DescribeStack(c, stackName)
```

Comment:
```suggestion
func DeleteStackIfExists(t *testing.T, c *cfn.Client, stackName string) error {
	t.Helper()      
	_, err := DescribeStack(c, stackName)
```
there's a linter for this call `thelper`

Reply:
updated and added `thelper` to linters